packages/aws-cdk-lib/aws-codepipeline-actions/test/codepipeline/invoke-action.test.ts

```diff
@@ -0,0 +1,76 @@
+import { Template, Match } from '../../../assertions';
+import * as codebuild from '../../../aws-codebuild';
+import * as codepipeline from '../../../aws-codepipeline';
+import * as s3 from '../../../aws-s3';
+import { App, Stack } from '../../../core';
+import * as cpactions from '../../lib';
+import { RevisionType } from '../../lib';
```

Comment:
Could be merged.

Reply:
@hwum 

Could you apply this one last thing? I will then approve it.

```suggestion
import * as cpactions from '../../lib';
```

And change the code with `cpactions.RevisionType`.